stdlib/public/core/Unicode.swift

```diff
@@ -705,8 +705,8 @@ public func transcode<
 
   var inputDecoder = inputEncoding.init()
   var hadError = false
-  for var scalar = inputDecoder.decode(&input);
-          !scalar.isEmptyInput();
+  for var scalar = inputDecoder.decode(&input)
+          !scalar.isEmptyInput()
```

Comment:
Good catch. Updated and running tests now. Will push up squashed commit once the tests pass.